Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DummyWalletInfo.kt

```diff
@@ -0,0 +1,105 @@
+package de.rki.coronawarnapp.ccl.dccwalletinfo.model
+
+// DO delete file
```

Comment:
USE TODO instead of DO? Then Android Studio can show all open TODOs.

Reply:
Yes :D , sonar does not like many `TODO` s which led to `CI` fail  🤷‍♂️

Reply:
Ah, I suspected something like this ;)